build.rs

```diff
@@ -13,7 +13,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     tonic_build::configure()
         .build_client(true)
         .build_server(false)
-        .format(true)
```

Comment:
I think we can leave it here. I will simply save new version of grpc stubs. After initial shock of how many files have been changed I am ok with it now :)

Reply:
format has been removed in latest version